src/snowflake/cli/_plugins/connection/commands.py

```diff
@@ -309,13 +309,13 @@ def test(
         schema = conn.schema
 
         if conn.role:
-            om.use(object_type=ObjectType.ROLE, name=f'"{conn.role}"')
+            om.use(object_type=UseObjectType.ROLE, name=f'"{conn.role}"')
```

Comment:
any reason we switched to UseObjectType? 
We don't need to link the object type itself to how it is used.

Reply:
Why are we changing ObjectManager calls though? We shouldn't do this right now.